cmd/addvalidators.go

```diff
@@ -180,8 +189,8 @@ func builderRegistration(secret tbls.PrivateKey, pubkey tbls.PublicKey, feeRecip
 	}, nil
 }
 
-// loadClusterState returns the cluster manifest from the given file path.
-func loadClusterState(conf addValidatorsConfig) (*manifestpb.Cluster, error) {
+// loadClusterManifest returns the cluster manifest from the given file path.
+func loadClusterManifest(conf addValidatorsConfig) (*manifestpb.Cluster, error) {
```

Comment:
Add TODO to load existing `cluster-manifest.pb` file.

Reply:
This TODO is already written in the `manifest.Load` function